src/config/cfg_rules.ini

```diff
@@ -332,6 +336,58 @@ option = scope
 option = users
 option = groups
 
+# Prompting during authentication
+[rule/allowed_prompting_password_options]
+validator = ini_allowed_options
+section_re = ^prompting/password$
+
+option = timeout
+option = debug
+option = debug_level
+option = debug_timestamps
+option = debug_microseconds
+option = debug_to_files
+option = command
+option = reconnection_retries
+option = fd_limit
+option = client_idle_timeout
+option = description
+option = password_prompt
+
+[rule/allowed_prompting_2fa_options]
+validator = ini_allowed_options
+section_re = ^prompting/2fa$
+
+option = timeout
+option = debug
+option = debug_level
+option = debug_timestamps
+option = debug_microseconds
+option = debug_to_files
+option = command
+option = reconnection_retries
+option = fd_limit
+option = client_idle_timeout
+option = description
+option = single_prompt
+option = first_prompt
+option = second_prompt
```

Comment:
Hi,

same as for `[prompting/password]`, only the options listed in the sssd.conf man page are supported, i.e. `single_prompt`, `first_prompt` and `second_prompt`.

bye,
Sumit